cmd/buildctl/debug/info.go

```diff
@@ -0,0 +1,43 @@
+package debug
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	bccommon "github.com/moby/buildkit/cmd/buildctl/common"
+	"github.com/urfave/cli"
+)
+
+var InfoCommand = cli.Command{
```

Comment:
Why is the command needed? We already have `version` command. Is there a valid use-case?

Reply:
We don't have a version command, only the `--version` flag. I think `info` is only for debug purposes like the `workers` one.

Reply:
If we add this also to `WorkerRecord` then it can just be shown on exiting commands. Trying to avoid adding new things if there isn't a clear need for them.